android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginViewModel.kt

```diff
@@ -149,6 +176,20 @@ constructor(
     )
   }
 
+  fun runSync() =
```

Comment:
do we need this? since we are calling practitioner data separately now?

Reply:
removed